keyboards/kc60/keymaps/noroadsleft/rules.mk

```diff
@@ -0,0 +1,2 @@
+// https://github.com/qmk/qmk_firmware/issues/3448#issuecomment-406636125
```

Comment:
Makefiles use `#` for comments, IIRC

Reply:
Huh, it's never warned me that this is wrong when I've compiled. But you're right.